packages/api/tests/.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "no-param-reassign": "off"
```

Comment:
Do you know off-hand why this is required? Seems like a good idea not to mutate parameters if we can avoid it.

Reply:
Yeah in the tests setting t.context throws eslint errors. This eslintrc file is just for the tests.

Reply:
👍

Reply:
I think we should turn it off across the repo. Reassign params in java script is a very common trend.